src/architecture/serialization.md

```diff
@@ -100,55 +102,83 @@ include!(concat!(env!("OUT_DIR"), "/example_mod.rs"));
 use exonum::proto::schema::*;
 ```
 
+Then to access a structure from a protobuf generated file, you need to indicate
+the name of the module with the generated files and the name of the required
+file and structure in it:
+
+```rust
+module_name::proto_file_name::StructNameInProtoFile
+```
+
+For example, the generated `Wallet` structure, included in the
+`cryptocurrency.proto` file, which resides in the [`proto`][module] module,
+will be available using `proto::cryptocurrency::Wallet`.
+
+`exonum_build` also includes a function which returns the path to the `.proto`
+files come with the `exonum` crate - `get_exonum_protobuf_files_path()`:
+
+```rust
+let exonum_protos = get_exonum_protobuf_files_path();
+    protobuf_generate(
+        "src/proto",
+        &["src/proto", &exonum_protos],
+        "example_mod.rs",
+    );
+```
+
+After applying the `get_exonum_protobuf_files_path()` function, users can
```

Comment:
I think this function is called rather than applied since there are no arguments